android/build.gradle

```diff
@@ -28,7 +28,7 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 24 // 16
+        minSdkVersion  21
```

Comment:
❤️

Reply:
Thank you @Alezhka for your Review on the future Flutter Sound V6.0. 👍 

I hope that V6.0 will be a great major FS version. 😄 
It will have support to **Record to a Flutter Stream** on ios and Android, and **playback from a Flutter Stream** on Android which is a feature that many, many, many Flutter Sound users are waiting for.

I will try also to support Flutter Sound on the Web Apps and Macos Desktop. But the work is actually not done.

Take care.

Reply:
Good luck!

We are waiting for the release ☺️